ui/src/app/services/interceptor/interceptor.service.ts

```diff
@@ -10,8 +10,10 @@ export class InterceptorService implements HttpInterceptor {
   count: number = 0
   constructor(private loader: LoaderService) {}
   intercept(req: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
-    this.loader.startLoader()
-    this.count++
+    if (req.url.includes('/connect')) {
```

Comment:
the loader keeps on loading even after database is connected: https://drive.google.com/file/d/1fQM61eP_t43wnJd0nQMk7vurWp8gt88z/view?usp=sharing&resourcekey=0-M5NvbEfU-pgECOqYE56i7w

Reply:
Bug fixed